src/Components/Calculator.js

```diff
@@ -1,43 +1,99 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
- [ ] Please, consider removing the linter disables, for the linter makes its exact job - support you to find the bugs in the early stages of building the app and fix them promptly before merging to the main branch 👍